scripts/mvftoms.py

```diff
@@ -0,0 +1,706 @@
+#!/usr/bin/env python
+
+################################################################################
+# Copyright (c) 2011-2018, National Research Foundation (Square Kilometre Array)
+#
+# Licensed under the BSD 3-Clause License (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy
+# of the License at
+#
+#   https://opensource.org/licenses/BSD-3-Clause
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# Produce a CASA-compatible Measurement Set from a KAT-7 HDF5 file (versions
+# 1 and 2) or a MeerKAT Visibility Format dataset (versions 3 and 4) using
```

Comment:
I would just call all of them MVF (versions 1 to 4). No need to distinguish KAT-7.